packages/js/src/ui/components/Header/StatusOptions.tsx

```diff
@@ -61,18 +60,19 @@ export const StatusItem = (props: {
   const style = useStyle();
 
   return (
-    <Popover.Close onClick={props.onClick}>
-      <button class={style('inboxStatus__dropdownItem', dropdownItemClasses())}>
-        <span class={style('inboxStatus__dropdownItemLabelContainer', dropdownItemLabelContainerClasses())}>
-          <span class={style('inboxStatus__dropdownItemLeftIcon', '')}>{props.icon()}</span>
-          <span class={style('inboxStatus__dropdownItemLabel', dropdownItemLabelClasses())}>{props.label}</span>
+    <Dropdown.Item
+      class={style('inboxStatus__dropdownItem', cn(dropdownItemVariants(), 'nt-flex nt-gap-8'))}
+      onClick={props.onClick}
+    >
+      <span class={style('inboxStatus__dropdownItemLabelContainer', 'nt-flex nt-gap-2 nt-items-center')}>
+        <span class={style('inboxStatus__dropdownItemLeftIcon')}>{props.icon()}</span>
+        <span class={style('inboxStatus__dropdownItemLabel')}>{props.label}</span>
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** The label renders in white color, add `nt-foreground` class to this‏

Reply:
We need to add `nt-foreground` top level, so everything inherits, no ?